### ISSUE TITLE
deprecate_warn("aw_get_metrics(arg)")

### ISSUE THREAD

**Issue by XoliloX** [benrwoodard/adobeanalyticsr#143]
Dear Ben,

I assume this error is not related to the package, but after some research, I'm unable to get to the bottom of it.
Basically, calling `aw_freeform_table()` gives me this:

```
Internal error: what must refer to arguments in the LHS of `=`.

  # Good:
  deprecate_warn("aw_get_metrics(arg = )")

  # Bad:
  deprecate_warn("aw_get_metrics(arg)")
```

The only fonction I was able to run without that error was the `aw_anomaly_report`. Would you be able to help me out?

In advance, thanks a lot for your answer!

Best,
D.

System : R Studio 1.4.1103, Mac OS 10.15,

**Comment by XoliloX** [comment]
Dear Charlie,

Thanks for commenting! So there's nothing I can do on my end to pull data before this is solved?

Thanks,
D.

**Comment by XoliloX** [comment]
Hi Charlie, thanks a lot ! 

Sure, but sorry for the newbie question; is it a package version or another package I should use for the dev one?

Thanks,
D.

**Comment by benrwoodard** [comment]
I'm not able to reproduce the error either on my side.  I've never seen a warning cause and error like this.  I'll merge the update today and let you know when it is done and how to install the dev version of the package using the 'devtools' package.

**Comment by benrwoodard** [comment]
@charlie-gallagher Yes!  Thanks for doing that.  Let us know @XoliloX what you find. Make sure to restart your R session once you install the dev version of the package.

**Comment by XoliloX** [comment]
Hi Ben, thanks ! I get the following error (translated):

```
Warning message:
‘RCurl’ package was compiled with R version 4.0.5 
Error in library(httr) : no package named  ‘httr’ was found
Execution stopped
Warning message:
In i.p(...) : installation of package
‘/var/folders/sc/46j_rwx94g36925rdqpnm42m0000gn/T//Rtmpvnf1gN/file755264e376c/httr_1.4.3.9000.tar.gz’ had non-zero exit status
```

**Comment by XoliloX** [comment]
Hi Ben, I fixed it ; R version was 4.0.5 was the problem. It's all working well now, sorry for the disturbance! Thanks for your help! Best, D.
